apps/calendar/js/common/calc.js

```diff
@@ -1,7 +1,7 @@
 define(function(require, exports) {
```

Comment:
Moved some stuff that's potentially used in both the frontend and backend to `common/`